### DISCUSSION TITLE
Standard Notes issues

### DISCUSSION THREAD

**Discussion by CyberSmurfar** [standardnotes/forum#3368] [General]
Hi

I dont know whats wrong but I had two accounts before on SN and now I have deleted one of them but now when I log in to my accountI have left I get this message, "Unable to write to local database. Please restart the app and try again". What means this message and why do I get this now?

**Comment by effieeee** [comment]
What app platform are you using at the moment? (desktop, web, or mobile)
Also, do you see this error when you try to sign in on any other device?

**Reply by CyberSmurfar** [reply]
Hi

I got the message when I used my laptop but now it seems it has vanished cause now I can log in to SN as usual. I had two accounts before but now I have deleted one of them and maybe it was therefore I got that message.
